cards/ecg-right-posterior-leads/card.md

```diff
@@ -52,3 +52,6 @@ Hints of an associated posterior infarct:
 2.  Upright T wave waves (in later stage)
 
 ## References
+
+1. [Edhouse J, Brady WJ, Morris F. ABC of clinical electrocardiography: Acute myocardial infarction-Part II. BMJ. 2002; 324: 963-6.] (https://doi.org/10.1136/bmj.324.7343.963)
```

Comment:
I actually just changed the rule. And then used an intermediate step with remark to rewrite the all the card files to comply to the new rule.

It was a piece of cake with Remark. Awesome package.